src/pTSL.py

```diff
@@ -76,27 +113,97 @@ def val_corpus_helper(self, corpus_file):
             input_strings.append(' '.join(map(str, string)))
         return dict(zip(input_strings, cond_probs))
 
+    def eval_conditional_probabilities(self, corpus_file,
+                                       verbose=False,
+                                       train=False,
+                                       corpus_filename=None,
+                                       projection_filename=None):
+        '''
+            evaluate conditional probabilities given corpus data.

```

Comment:
I'd change this to something like `Calculates conditional probabilities of corpus forms based on the current pTSL grammar.`